### PR TITLE
Add hello-unity app

### PR DESCRIPTION
**Pull request by mhall119**
Add's the hello-unity app

### CODE REVIEW

**Comment by dholbach** [comment]
Can you follow https://github.com/ubuntu/snappy-playpen/blob/master/CONTRIBUTING.md?

**Comment by didrocks** [comment]
I think you should follow the contributing guide as per PR message

File: hello-unity/snapcraft.yaml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    source-type: bzr
+    files:
+      hello-unity: opt/bin/hello-unity
+      hello-unity.svg: usr/share/pixmaps/hello-unity.svg
+      hello-unity-mono-light.svg: usr/share/pixmaps/hello-unity-mono-light.svg
+      hello-unity-attention.svg: usr/share/pixmaps/hello-unity-attention.svg
+      hello-unity-mono-dark.svg: usr/share/pixmaps/hello-unity-mono-dark.svg
+
+  patches:
+    plugin: quilt
```
Comment (dholbach) [inline hello-unity/snapcraft.yaml]: Is this a local plugin that's not part of this commit?

**Comment by dholbach** [comment]
It looks like you added `kcalc` to this branch too. Can you fix that?

**Comment by dholbach** [comment]
The build fails now too - thanks @didrocks !

**Comment by didrocks** [comment]
First time ever someone thanks me for a build failing! :-) Happy that it works now!

File: hello-unity/hello-unity.wrapper [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+if [ ! -e $GDK_PIXBUF_MODULE_FILE ]; then
+  $SNAP/usr/lib/$ARCH/gdk-pixbuf-2.0/gdk-pixbuf-query-loaders > $GDK_PIXBUF_MODULE_FILE
+fi
+
+# Keep an array of data dirs, for looping through them
+IFS=':' read -r -a data_dirs_array <<< "$XDG_DATA_DIRS"
+
+# Setup compiled gsettings schema
+GS_SCHEMA_DIR=$XDG_DATA_HOME/glib-2.0/schemas
```
Comment (rednoah) [inline hello-unity/hello-unity.wrapper]: Took me forever to figure out all this stuff myself. Hopefully this will implicitly handled by snapcraft, or not be necessary at with improvements to snapd.

**Comment by dholbach** [comment]
Can we close this one?

**Comment by dholbach** [comment]
I will close this one for now. Please reopen once this has been worked out.
